src/test/java/de/rabea/game/GameRunnerTest.java

```diff
@@ -41,28 +38,22 @@ public void creates3x3Board() {
         GameRunner gameRunner = new GameRunner(new JavaFXUi(viewUpdaterSpy), new PlayerFactory(null));
         Board board = gameRunner.createBoard(3);
 
-        TestCase.assertEquals(3, board.getDimension());
-    }
-
-    @Test
-    public void creates4x4Board() {
-        GameRunner gameRunner = new GameRunner(new JavaFXUi(viewUpdaterSpy), new PlayerFactory(null));
-        Board board = gameRunner.createBoard(4);
-
-        TestCase.assertEquals(4, board.getDimension());
+        assertEquals(3, board.getDimension());
     }
 
     @Test
     public void showsOptionsForGameMode() {
         GameRunner gameRunner = new GameRunner(new JavaFXUi(viewUpdaterSpy), new PlayerFactory(null));
         gameRunner.displayGameModeOptions();
+
         assertTrue(viewUpdaterSpy.hasShownGameModeOptions);
     }
 
     @Test
-    public void createsANewGame() {
+    public void createsNewGameWithGuiPlayerAndUnbeatablePlayer() {
         GameRunner gameRunner = new GameRunner(new JavaFXUi(viewUpdaterSpy), new PlayerFactory(null));
```

Comment:
This is a tough one!
Checking for `x != null` makes sense IF there is a situation where `x == null` will hold.
Testing that you get something back that _a kind of Game_ is already guaranteed by the compiler.
Now, testing that the players that get back are of the right _type_, feels like I am testing the _factory_. 
I am asserting that `GameMode.GuiHumanVsComputer` gives me a `HumanPlayer` and a `UnbeatableComputerPlayer`, right?

I am not really sure how to test that 'a correct game is constructed' though...